src/Common/Entity/Style/Style.php

```diff
@@ -383,6 +389,28 @@ public function hasSetWrapText(): bool
         return $this->hasSetWrapText;
     }
 
+    public function textRotation(): int
+    {
+        return $this->textRotation;
+    }
+
+    /**
+     * @param int $rotation Rotate text
+     */
+    public function setTextRotation(int $rotation = 0): self
+    {
+        $this->textRotation = $rotation;
+        $this->hasSetTextRotation = true;
+        $this->isEmpty = false;
+
+        return $this;
+    }
+
+    public function hasSetTextRotation(): bool
+    {
+        return $this->hasSetTextRotation;
```

Comment:
`0` seems to be the default value: can we remove the `hasSetTextRotation` property and just check if `0 !== $this->textRotation`?

Reply:
Yes.